cypress/support/network_commands.js

```diff
@@ -6,17 +6,21 @@
  */
 
 Cypress.Commands.add('importJSONMapping', (filepath, hostname = 'localhost', port = '9200') => {
+  const parseIndex = (indexValue) => {
```

Comment:
use `object destructuring` to simplify code

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring

Reply:
I added the object destructuring to `parseIndex` and `parseDocument`, thank you for the suggestion it definitely makes the code cleaner!